### PR TITLE
Swerve Optimization

### PR DESCRIPTION
**Pull request by centiwati**
Adding CANBus optimization to the swerve modules.

### CODE REVIEW

File: src/main/java/frc/robot/generated/TunerConstants.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package frc.robot.generated;
 
 import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.hardware.TalonFX;
```
Comment (Jnesselr) [inline src/main/java/frc/robot/generated/TunerConstants.java:4]: This import appears unused.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -67,6 +67,7 @@ public RobotContainer() {
     configureBindings();
     shooterSubsystem.shooterInit();
     intakeSubsystem.intakeInit();
+    drivetrain.optimizeCan();
```
Comment (Jnesselr) [inline src/main/java/frc/robot/RobotContainer.java:72]: Should we be consistent? This makes it apparent the subsystem name for drivetrain doesn't match the others and neither does this specific method call.

**Review by Jnesselr — APPROVED** [review]
Most of my comments are nitpicks

File: src/main/java/frc/robot/subsystems/SwerveSubsystem.java [hunk 3]
```diff
@@ -39,6 +41,13 @@ public Command applyRequest(Supplier<SwerveRequest> requestSupplier) {
         return run(() -> this.setControl(requestSupplier.get()));
     }
 
+    public void optimizeCan() {
+        for (int i = 0; i > 3; i ++) {
```
Comment (GucciBanana11) [inline src/main/java/frc/robot/subsystems/SwerveSubsystem.java]: This for loop won't run, the second parameter needs to be flipped

**Review by GucciBanana11 — CHANGES_REQUESTED** [review]
Everything else looks good except the for loop

File: src/main/java/frc/robot/subsystems/SwerveSubsystem.java (hunk 3 above)
Comment (Jnesselr) [reply src/main/java/frc/robot/subsystems/SwerveSubsystem.java]: Good catch!

**Review by GucciBanana11 — APPROVED** [review]
This code looks really good! Whoever wrote it must be a genius!

**Pull request merged**